ert/model/drogon_ahm.ert

```diff
@@ -180,8 +180,7 @@ FORWARD_MODEL ECLCOMPRESS  -- Compress eclipse include files
 -- FORWARD_MODEL INSERT_NOSIM
 -- FORWARD_MODEL ECLIPSE100(<VERSION>=2022.2)
 
-FORWARD_MODEL OPM_FLOW  -- flowrc23
-
+FORWARD_MODEL FLOW(<VERSION>=rc23, <OPTS>="--solver-max-time-step-in-days=31.0")
```

Comment:
What is the syntax for adding multiple options? Should we consider adding a second one for illustration purpose?

Reply:
It is mentioned in the docs that they are to be space-separated: https://fmu-docs.equinor.com/docs/ert/reference/configuration/forward_model.html#FLOW